estimage/persistence/card/abstract.py

```
@@ -0,0 +1,68 @@
+
+
+class CardSaver(abstract.Saver):
+    WHAT_IS_THIS = "card"
+    @classmethod
+    def bulk_save_metadata(cls, cards: typing.Iterable[data.BaseCard]):
+        saver = cls()
+        for t in cards:
+            t.pass_data_to_saver(saver)
+
+    @classmethod
+    def bulk_save_metadata(cls, cards: typing.Iterable[data.BaseCard]):
```

Ruff (0.13.1)

Redefinition of unused `bulk_save_metadata` from line 59

(F811)